# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1362036344 18000
#      Thu Feb 28 02:25:44 2013 -0500
# Node ID 236be61797850ab411dd6fe6ebf0837b08ede631
# Parent  aa5e1e8dce6676387964bc7c108c0e8ebbe5c55c
# Parent  8a8e04aa3c98f81e787a3b128796633127c71e73
maint: periodic merge of stable to default

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -70,8 +70,9 @@ 551566201318bf615b27c60ccf9368f4844008bd
 a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
 2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
 df1aceb8f0bc6b5b5062907931cc663467f57d93 ss-3-7-1
 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c rc-3-6-4-1
 faefa1bea8ddae3cab170afdeab68d3d15c4e623 ss-3-7-2
 0000000000000000000000000000000000000000 ss-3-7-2
 23a7661e529ae9bfc91693618f8c314c31f695ca ss-3-7-2
 cc5a7d1233f3acea85648baeb754fc0e8f225225 rc-3-6-4-2
+b29b10fbb7448cdfe29322446e1a589e7fe1a40a release-3-6-4
